### ISSUE TITLE
checked property not toggling on <input>

### ISSUE THREAD

**Issue by Nfinished** [minhur/bootstrap-toggle#111]
`this.$element.prop('checked', true);` and its counterpart in the off function do nothing to modify the state of the <input> tag. Admittedly I haven't checked to see if this affects submission, but how could it not?

**Comment by msct** [comment]
I agree this should be addressed.  Keeping the 'checked' property synced should be a function of the plugin, not user code using events.  A quick look and this is likely the underlying problem for #94 and #105 as well.

**Comment by Bulld0z3r** [comment]
+1, this should be solved!

**Comment by milmih** [comment]
Would you say that #140 is a dup of this issue?

**Comment by Nfinished** [comment]
@minhur seems so.

**Comment by milmih** [comment]
I'll the maintainer close it.
